dev-util/bats/bats-1.2.1-r1.ebuild

```
@@ -0,0 +1,38 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PN="bats-core"
+DESCRIPTION="Bats-core: Bash Automated Testing System"
+HOMEPAGE="https://github.com/bats-core/bats-core/"
+SRC_URI="https://github.com/${MY_PN}/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~ppc64 ~x86"
+
+IUSE="threads"
+
+DEPEND="
+	app-shells/bash:*
+	!ppc64? ( threads? ( sys-process/parallel ) )
```

```suggestion
	ppc64? ( sys-process/parallel )
```
Since `threads` doesn't actually do anything in this ebuild.

---

switched to postinst printing only